roles/kerberos/defaults/main.yml

```diff
@@ -9,3 +9,4 @@ kerberos:
   default_tgs_enctypes: aes256-cts-hmac-sha1-96 aes128-cts-hmac-sha1-96 arc-four-hmac rc4-hmac
   permitted_enctypes: aes256-cts-hmac-sha1-96 aes128-cts-hmac-sha1-96 arc-four-hmac rc4-hmac
   canonicalize: true
+  allow_weak_crypto: false
```

Comment:
Thanks @utkarsh5474 for taking care of this. Do we want to run doc.py to document this additional var in VARIABLES.md ?

Reply:
No Sneha, these set of kerberos var don't go in variables.md. 
I'll raise a doc ticket for Alex where we can mention as a footnote to use this var if required.

Reply:
got it, thanks for clarifying